src/index.js

```diff
@@ -35,26 +42,28 @@ if (utils.isIE11) {
 var error = debug('A-Frame:error');
 var warn = debug('A-Frame:warn');
 
-if (window.document.currentScript && window.document.currentScript.parentNode !==
-    window.document.head && !window.debug) {
-  warn('Put the A-Frame <script> tag in the <head> of the HTML *before* the scene to ' +
-       'ensure everything for A-Frame is properly registered before they are used from ' +
-       'HTML.');
-}
-
-// Error if not using a server.
-if (window.location.protocol === 'file:') {
-  error(
-    'This HTML file is currently being served via the file:// protocol. ' +
-    'Assets, textures, and models WILL NOT WORK due to cross-origin policy! ' +
-    'Please use a local or hosted server: ' +
-    'https://aframe.io/docs/0.5.0/introduction/getting-started.html#using-a-local-server.');
+if (!window.cordova) {
+  if (window.document.currentScript && window.document.currentScript.parentNode !==
+      window.document.head && !window.debug) {
+    warn('Put the A-Frame <script> tag in the <head> of the HTML *before* the scene to ' +
+        'ensure everything for A-Frame is properly registered before they are used from ' +
+        'HTML.');
+  }
+
+  // Error if not using a server.
+  if (window.location.protocol === 'file:') {
+    error(
+      'This HTML file is currently being served via the file:// protocol. ' +
+      'Assets, textures, and models WILL NOT WORK due to cross-origin policy! ' +
+      'Please use a local or hosted server: ' +
+      'https://aframe.io/docs/' + pkg.version + '/introduction/installation.html#use-a-local-server.');
+  }
 }
 
 require('present'); // Polyfill `performance.now()`.
 
 // CSS.
-if (utils.device.isBrowserEnvironment) {
+if (utils.device.isBrowserEnvironment || window.cordova) {
```

Comment:
For some reason isBrowserEnvironment is `false` when running in Cordova. I can try to fix this, or we can just add a check here as I have done.